modules/client-configuration/pages/channels.adoc

```diff
@@ -42,3 +42,15 @@ In most cases, you will need to install the base channel for your distribution,
 
 If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
 For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].
+
+
+
+== Unified Installer Updates Channels on {sle} Clients
+
+All {sle} clients require access to an installer updates channel.
+This channel is used by the unified installer to ensure it is up to date before it installs the operating system.
+
+For {sles} clients the installer updates channel is installed by default when you add a product that contains them, and are enabled when you first synchronize the client.
+
+For all other {sle} variants, including {sle} for SAP,  you must add the installer updates channel manually.
+To do this, clone the installer updates channel that belongs to the appropriate base channel, and assign it to the client.
```

Comment:
```suggestion
To do this, clone the appropriate {sles} installer updates channel below the base channel of these {sle} variants.
When creating a autoinstallation distribution for these {sle} variants after the channel was cloned , it will be used automatically.
```